### ISSUE TITLE
Extract Redis from Posts Service to be shared via Satellite

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#1908]
In #1838 we're adding a Posts Service, and it includes Redis code for connecting to a real Redis server, as well as a mock instance.  Let's refactor this a bit:

1. Redis access is going to be needed by various services, so let's move it out of Posts and put it in Satellite.  We should be able to do something like this:
```js
const { Redis } = require('@senecacdot/satellite');
...
// Get an instance of a Redis client, pre-configured for us to use our server
const client = Redis();
```
2. The `MOCK_REDIS=1` stuff can either be replicated in Satellite too, or we could also expose a `MockRedis` function to work similar to `Redis`, and people could use that in tests.  I'm not sure which is better.  It might become obvious once we try to write tests with this.

**Comment by Metropass** [comment]
I could begin the port of Redis into Satellite

**Comment by Metropass** [comment]
In the PR you linked, there's a line of code that accesses an Environment Variable in Telescope.
```js
const useMockRedis = process.env.MOCK_REDIS;
```
If we're gonna port this into Satellite, could I just pass the variable as an argument for our Redist module?

**Comment by humphd** [comment]
I don't think so: most of the code should be unaware of this happening, whereas making it an argument forces all code to understand it.

**Comment by HyperTHD** [comment]
Closing since [Satellite PR #11](https://github.com/Seneca-CDOT/satellite/pull/11) successfully merges redis into satellite and is available as of ver 1.10.0.
